internal/storage/storage_handle.go

```diff
@@ -77,6 +94,18 @@ func NewStorageHandle(ctx context.Context,
 		}),
 		storage.WithPolicy(storage.RetryAlways))
 
-	sh = &storageClient{client: sc}
+	client := &storageClient{client: sc}
+	return client, err
+}
+
+func (sh *storageClient) BucketHandle(bucketName string) (bh *bucketHandle,
+	err error) {
```

Comment:
dont break the function definitions inbetween. Put the entire definition on one line. https://github.com/golang/go/wiki/CodeReviewComments#line-length